packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -28,6 +28,7 @@ RUN dnf -y install dnf-plugins-core epel-release && \
                    python3-coveralls \
                    python3-requests \
                    python3-docopt \
+                   tcpreplay \
```

Comment:
I guess the Fedora container spec needs to be updated, too. It is not in the CI anymore but still in the packaging directory.

Reply:
Oops. You are right.

Reply:
Done, thanks!